fili-core/src/main/java/com/yahoo/bard/webservice/util/TableUtils.java

```diff
@@ -36,8 +36,7 @@
     ) {
         return Stream.<Stream<String>>of(
                 getDimensions(request, query)
-                        .map(Dimension::getApiName)
-                        .map(table::getPhysicalColumnName),
+                        .map(Dimension::getApiName),
```

Comment:
According to the JavaDoc, this method returns fact-store column names (i.e. physical column names). Note that the `AggregatableDimensionsMatcher::test` expects it to return fact-store column names. But it looks like we're now returning api names.